components/ADatePicker.vue

```diff
@@ -287,10 +292,29 @@ const onOpen = () => {
     }
   }, 100);
 };
+
+const onClose = () => {
```

Comment:
```suggestion
const close = () => {
```
C'est plutot une action qu'on fait ici, plutot qu'un évenement